File: CS/dxSample/Custom Tree List/CustomTreeListViewInfo.cs
using System;
using System.Collections.Generic;
using System.Drawing;
using System.Linq;
using System.Text;
using DevExpress.XtraTreeList;
using DevExpress.XtraTreeList.ViewInfo;

namespace dxSample {
    class CustomTreeListViewInfo : TreeListViewInfo {
        public CustomTreeListViewInfo(TreeList tl ):base(tl) {
        }
        protected override Rectangle CalcRowFooterInfo(RowFooterInfo fInfo, System.Drawing.Rectangle bounds, bool isLast = false) {
            int lvl = fInfo.Node.Level;
            if (lvl == 2)
                return bounds;
            return base.CalcRowFooterInfo(fInfo, bounds, isLast);
        }
    }
}

File: VB/dxSample/Custom Tree List/CustomTreeListViewInfo.vb
Imports System
Imports System.Collections.Generic
Imports System.Linq
Imports System.Text
Imports DevExpress.XtraTreeList
Imports DevExpress.XtraTreeList.ViewInfo

Namespace dxSample
    Friend Class CustomTreeListViewInfo
        Inherits TreeListViewInfo

        Public Sub New(ByVal tl As TreeList)
            MyBase.New(tl)
        End Sub
        Protected Overrides Sub CalcRowFooterInfo(ByVal rowIndex As Integer, ByVal node As DevExpress.XtraTreeList.Nodes.TreeListNode, ByVal rowArgs As CalcRowGroupInfoArgs, ByVal even As Boolean)
            Dim lvl As Integer = node.Level
            If lvl = 2 Then
                Return
            End If
            MyBase.CalcRowFooterInfo(rowIndex, node, rowArgs, even)
        End Sub
    End Class
End Namespace
